homeassistant/core.py

```diff
@@ -2019,10 +2019,36 @@ def services(self) -> dict[str, dict[str, Service]]:
     def async_services(self) -> dict[str, dict[str, Service]]:
```

Comment:
This isn't actually used in production code anymore, only tests. We might eventually be able to remove it